src/webview/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -298,18 +320,25 @@ private void setUpUiClient(WebView container) {
 				return true;
 			}
 			@Override public void onGeolocationPermissionsShowPrompt(final String origin, final GeolocationPermissions.Callback callback) {
+				trace(this, "onGeolocationPermissionsShowPrompt() :: origin=%s, callback=%s", origin, callback);
```

Comment:
The callback doesn't have a nice toString so you end up just getting a memory reference which isn't very helpful. Just remove that from the trace to keep the logs more readable.

Reply:
Yes I see the callback `toString` method is the `Object` implementation that only gets you the object id, and the origin string is always the URL set by the user so doesn't make sense.

Reply:
Removed.